### PR TITLE
Bluepyopt new

### PR DESCRIPTION
**Pull request by adevress**

### CODE REVIEW

File: bbp/nse/bluepyopt/default.nix [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  fetchFromGitHub,
+  pythonPackages
+}:
+
+pythonPackages.buildPythonPackage rec {
+    pname = "BluePyOpt";
+    version = "1.6dev";
+    name = "${pname}-${version}";
+
+    src = fetchFromGitHub {
```
Comment (arsenius7) [inline bbp/nse/bluepyopt/default.nix]: why not fetchPypi?

Comment (adevress) [reply bbp/nse/bluepyopt/default.nix]: I tend to trust more github than anything else to keep old version and archive properly.  Do you see it as an issue ? Is there version released on Pypi and not tagged on github ?

Comment (arsenius7) [reply bbp/nse/bluepyopt/default.nix]: I see an issue (even if a minor one) with using two approaches for apparently same thing.
For `fetchPypi` one we have a benefit of not having to maintain version-commit correspondence manually and thus having guaranteed same versioning in virtualenv and Nix worlds, and I find it useful; but if trust in GitHub is more important, `fetchFromGitHub` should be preferred in other cases as well.

**Comment by arsenius7** [comment]
Otherwise LGTM, but I guess @wvangeit's opinion matters more here.

File: bbp/nse/bluepyopt/default.nix (hunk 1 above)
Comment (adevress) [reply bbp/nse/bluepyopt/default.nix]: Thx, I think I will keep Github for now and see with Werner for the testing

Comment (wvangeit) [reply bbp/nse/bluepyopt/default.nix]: Ok, but then at least update the version number to the correct one ...
i.e. 1.6.27

**Comment by wvangeit** [comment]
Mmm, so no fix to the version number before the merge ? ...

**Comment by adevress** [comment]
> Mmm, so no fix to the version number before the merge ? ...

I will happily fix it if you tell me which one you want. I would have done it before the merge if you commented before the merge ^^

**Comment by wvangeit** [comment]
Oh my, I guess all my comments are still 'pending'. Though that at least the comments would be immediately sent ...

**Comment by adevress** [comment]
Np that's happen. Let me send a second pull request with your fixes.

**Pull request merged**